content/04.results.md

```diff
@@ -2,42 +2,54 @@
-A similar trend was observed when predicting the GiG edge; however, the performance differences were small for this edge type making the importance difficult to assess.  
-The last column shows increasing performance (AUROC and AUPR) for both DaG and CtD when sampling from all label functions.
-CbG and GiG also had increased performance when one random label function was sampled, but performance decreased drastically as more label functions were added.
-It is possible that a small number of irrelevant label functions are able to overwhelm the distant supervision label functions in these cases (see Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
-
-### Random Label Function Generative Model Analysis
+We reported the results in AUROC and AUPR (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}; Supplemental Figures {@fig:auroc_gen_model_tune_set} and {@fig:auroc_gen_model_tune_set}).  
+Performance increases as the number of edge-specific label functions is added to the baseline model (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}). 
+The Compound treats Disease (CtD) edge type is a quintessential example of this trend.
+The baseline model starts off with an AUROC score of 52% and an AUPRC of 29%, then performance increases to 76% and 48% respectively as more CtD related label functions are included. 
+Gene interacts Gene (GiG), Disease associates Gene (DaG) and Compound binds Gene (CbG) edge types share a similar trend.
+Baseline performance starts off with an AUROC of 57% for DaG, 78% for CbG and 66% for GiG, then increases to 62% for DaG, 81% for CbG and 75% for GiG.
+Similarly with AUPR, the baseline starts off at 43% for DaG, 29% for CbG and 30% for GiG, then increases to 44%, 30% and 35% respectively.
+
+We based our initial hypothesis on the idea that certain edge types would share related linguistic features and terminology which would be amenable to label function transfer.
+For example, sentences for edges CbG and GiG typically describe physical interactions whether it be between a compound and a gene or a pair of two protein produced from genes.
+We observed performance increases in terms of both AUROC and AUPR when using label functions from the GiG edge type to predict CbG edges (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}).
+Likewise when using CbG related label function to predict the GiG edge type; however, in both cases GiG related label functions out performed the CbG related label functions.
+Regarding DaG, each label function source had similar performance when predicting the test set (Figures {@fig:auroc_gen_model_test_set} and {@fig:aupr_gen_model_test_set}); however the difference is more apparent when predicting the tune set (Supplemental Figures {@fig:auroc_gen_model_tune_set} and {@fig:auroc_gen_model_tune_set}).
```

Comment:
If you're going to bring in the tune set it'd be helpful for me to understand why. Is there an interpretation that the reader needs to take away here? Why not just focus on the test set, where we expect the estimate of performance to be accurate?